src/frontend/src/lib/components/ui/RoundedIcon.svelte

```diff
@@ -2,16 +2,10 @@
 	import type { ComponentType } from 'svelte';
 
 	export let icon: ComponentType;
-	export let iconSize = '2.9rem';
-	export let backgroundStyleClass = 'bg-brand-subtle-alt';
 	export let iconStyleClass = '';
-	export let additionalStyleClass = '';
 </script>
 
 <div class="relative">
-	<div
-		class={`rounded-full ${backgroundStyleClass}`}
-		style={`width: ${iconSize}; aspect-ratio: 1/1; ${additionalStyleClass}`}
-	/>
+	<div class="rounded-full bg-brand-subtle-alt" style={`width: 2.9rem; aspect-ratio: 1/1;`} />
```

Comment:
Since we are already there, Tailwind classes instead of style?

Reply:
yes!!! I was hoping for this eheh! done